source/api_documentation/transactions_api/_advertiser_optional_param_table.rst

```diff
@@ -0,0 +1,77 @@
+Additional Feature Parameters
+-----------------------------
+
+..  list-table::
+  :widths: 30 8 40
+  :header-rows: 1
+  :class: parameters
+
+  * - Field
+    - Name in Reports
+    - Description
+
+  * - advertiser_call_fee_localized
+    - Fees
+    - Advertiser Telecommunications fee associated with transaction
+
+  * - affiliate_campaign_id_from_network
+    - Affiliate Campaign ID
+    - The Campaign ID from the network as set on the affiliate campaign.
+
+  * - bundled_destination_ids
+    - Bundled Destination Ids
+    - 
+
+  * - bundled_destination_ids_from_network
+    - Bundled Destination Ids From Network
+    - 
+
+  * - call_outcome_custom_parameter_1
```

Comment:
OK. I can dig those up. 
But that leads me to ask why those descriptions don't show up with the technique we used? 
Shouldn't they be available?